src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleFucker.kt

```diff
@@ -226,12 +237,28 @@ object ModuleFucker : Module("Fucker", Category.WORLD, aliases = arrayOf("BedBre
         }
     }
 
+    private var spawnLocation: Vec3d? = null
+    private val gameStartHandler = handler<PacketEvent> {
+        if (it.packet is PlayerPositionLookS2CPacket) {
```

Comment:
The only thing that worries me is, when you get flagged this might cause your "spawn position" to shift away. But usually this should be alright most of the time.

Reply:
The flag usually does not cause a setback above 16 blocks